src/pint_pal/make_release.py

```diff
@@ -14,12 +14,12 @@
 import os
 import numpy as np
 from matplotlib import pyplot as plt
-import timing_analysis.lite_utils as lu
+import pint_pal.lite_utils as lu
 import pint.utils
 import astropy.units as u
 
 # accessible to functions here, apparently
-TA_PATH = "/home/jovyan/work/timing_analysis/" # assume running from here?
+TA_PATH = "/home/jovyan/work/pint_pal/" # assume running from here?
```

Comment:
Completely minor point, but if `TA` here stands for "timing analysis" is that a good enough reason to rename the variable?

Reply:
Probably a good idea. This might be harder to track down globally though, so if we miss some, we can fix them later. Good catch!

Reply:
I have renamed all instances of `TA_*` to `PAL_*`

~ Joe G.